src/assets/main.css

```diff
@@ -0,0 +1,21 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer utilities{
+    .width{
+        width: 600px;
+    }
+    .min-h-82{
+        min-height: 82px;
+    }
+
+    .min-h-163{
+        min-height: 163px;
+    }
+    .dropdown-toggle{
+    width: 125px;
```

Comment:
Please format this part in the same way how the other classes are and remove the empty line.